json-from-csv/handler.py

```diff
@@ -18,6 +18,20 @@ def run(event, context):
 
 # retrieve configuration from parameter store
 def get_config():
+    config =  {
+        "process-bucket-read-basepath": 'process',
+        "process-bucket-write-basepath": 'finished',
+        "image-server-bucket-basepath": '',
+        "manifest-server-bucket-basepath": '',
+        "sequence-csv": 'sequence.csv',
+        "main-csv": 'main.csv',
+        "canvas-default-height": 2000,
+        "canvas-default-width": 2000,
+        "notify-on-finished": "jhartzle@nd.edu",
+        "event-file": "event.json"
+    }
+
+    # read the keys we want out of ssm
```

Comment:
Could you also remove the MaxResults=4 in the paginator below?  No need to limit that.